content/engineering/guest-post/passwordless-authentication-for-your-php-project/index.md

```diff
@@ -0,0 +1,429 @@
+      $passwordLessLoginAPI = new PasswordLessLoginAPI();
+       $phone_number = $_POST['phone'];
+       $phone = $phone_number; //Required 
+       $smsTemplate = "smsTemplate"; //Optional
+       
+       $result = $passwordLessLoginAPI->passwordlessLoginByPhone($phone,$smsTemplate);
+          if($result){
+            $_SESSION['phone'] = $phone_number;
+            header("location:otp.php");
+            exit();
+    } else {
+        echo 'Unable to send verification code';
+    }
+          }
+?>
+```
+
+For `otp.php`, add the code as follows:
+
+```html
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title> Veification Page</title>
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+    />
+  </head>
+
+  <body>
+    <div class="d-flex min-vh-100 justify-content-center align-items-center">
+      <div class="shadow-sm mx-2 border rounded p-5">
+        <h3>Confirm it's you</h3>
+        <form action="otp.php" method="POST">
+          <div class="form-group">
+            <label for="uinput" class="mt-3 mb-1 text-muted">
+              An otp code has been sent to your phone
+            </label>
+            <input id="uinput" type="text" name="otp" class="form-control" />
+            <button name="submit" type="submit" class="btn btn-success mt-2 w-100" >
+              Submit
+            </button>
+          </div>
+        </form>
+      </div>
+    </div>
+  </body>
+</html>
+```
+
+For `otp_verify.php`, add the code as follows:
+
+```php
+<?php
+   session_start();
+   require_once 'config.php';
+   use \LoginRadiusSDK\CustomerRegistration\Authentication\PasswordLessLoginAPI;
+  if (isset($_POST['submit'])) {
+    $passwordLessLoginAPI = new PasswordLessLoginAPI();
+  $otp= $_POST['otp'];
+  $phoneid = $_SESSION['phone']; 
+  $payload = array(
+    "otp"  => $otp,
+    "phone" => $phoneid,
+); //Required 
+    $fields = null; //Optional 
+    $smsTemplate = "smsTemplate"; //Optional
+    
+    $result = $passwordLessLoginAPI->passwordlessLoginPhoneVerification($payload,$fields,$smsTemplate);
+    if($result){
+      $res = json_decode($result);
+      $phoneVerified = $res["Profile"]["PhoneIdVerified"];
+      if($phoneVerified){
+        header("location:homepage.php");
+    }else{
+        echo '<script>alert("incorrect otp")</script>';
+    }
+}
+?>
+```
+
+Almost done! Just create a homepage that says welcome back user. Create a file called `homepage.php` and add the code as follows:
+
+```php
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <h1>welcome back user</h1>
+</body>
+</html>
+```
+
+You can now run application and test it. That's how you can implement passwordless authentication using LoginRadius for your PHP project.
```

Comment:
@oyedeletemitope Please add a screenshot showing that the authentication is successful